man/it.man

```diff
@@ -0,0 +1,61 @@
+.TH VRMS-RPM 1
+.SH NAME
```

Comment:
This header looks like it's not translated.

Reply:
Sorry, I've missed it. Done.